cyberark/conjur
# Adds documentation for host factory "create tokens" route

Closes #212

#### What does this pull request do?
Documents how to create host factory tokens using the Conjur CE API

#### What background context can you provide?
Our best practices often suggest using a [Host Factory](https://developer.conjur.net/reference/services/host_factory), and we are currently working on updating our API clients, so I documented this feature.

#### Where should the reviewer start?
`apidocs/src/host_factory.md`

#### How should this be manually tested?
GIVEN I check out the linked branch
AND I generate the apidocs according tot he README,
WHEN I read the docs under the "host factory" heading,
THEN I seen an example of how to create a token,
AND I get an explanation of how to build the body of the request,
AND I see a response and possible error codes

#### Screenshots (if appropriate)
![host factory apidoc](https://user-images.githubusercontent.com/4389854/28988135-0c61ba0a-7934-11e7-87f2-7c66c6e693da.png)

#### Link to build in Jenkins (if appropriate)
https://jenkins.conjur.net/job/possum/job/doc%252Fhostfactory-tokens-create%252F212/
